superset/templates/superset/base.html

```diff
@@ -18,14 +18,18 @@
 #}
 {% extends "appbuilder/baselayout.html" %}
 {% set serverRoot =  request.script_root %}
+{% set walkMeEnabled =  request.environ['WALKME_ENABLED'] %}
 
```

Comment:
we can pick this variable directly from config like--
{% set walkMeEnabled = appbuilder.app.config.get('WALKME_ENABLED') %}

and we can avoid to set it in PrefixMiddleware env variable

Reply:
I was just following the already mentioned way to maintain consistency :) I have assigned it in variable first and then reading it back to assign in environment variable.